cowprotocol/explorer
# [Solvers] Add active solvers page

# Summary

Closes #221 

Added Solvers page along with active solvers table.

<img width="1090" alt="Screen Shot 2022-09-22 at 23 32 07" src="https://user-images.githubusercontent.com/11525018/191882447-be9c06b3-64d4-4043-920f-47f1b92dcd2c.png">


# To Test

1.  Open the page `solvers`
      * You'll see the active solvers tab with the data in it.

### Notes

- Settlements tab is not implemented so pagination refers only to active solvers table.
